website/static/css/custom.css

```diff
@@ -233,3 +233,15 @@ details[open] > summary {
 .tip-anchor::after {
     content: "∞"
 }
+
+blockquote {
```

Comment:
Could you provide before / after screenshots? So we can better understand what was changed instead of blindly merging

Reply:
Ah yes, pardon. This increases the text contrast of yellow blockquotes. The light gray text was quite light: https://i.imgur.com/B0iaAYI.png